File: ProblemSet/61A_Ultra-Fast Mathematician/C/solution.c
// Date : 30 - Sep - 2020 By Jatin Sharma

#include <stdio.h>
#include <string.h>

int main()
{
    int i;
    char a[105], b[105];
    scanf("%s", a);
    scanf("%s", b);

    for (i = 0; i < strlen(a); i++)
    {
        if (a[i] == '0' && b[i] == '0')
        {
            a[i] = '0';
        }
        else if (a[i] == '0' && b[i] == '1')
        {
            a[i] = '1';
        }
        else if (a[i] == '1' && b[i] == '0')
        {
            a[i] = '1';
        }
        else if (a[i] == '1' && b[i] == '1')
        {
            a[i] = '0';
        }
    }
    printf("%s\n", a);
    return 0;
}

File: ProblemSet/61A_Ultra-Fast Mathematician/Python/solution.py
# Date : 30-Sep-2020 By Jatin Sharma

n1 = list(map(int,input()))
n2 = list(map(int,input()))

result = []
length = len(n1)


for i in range(length):
    if n1[i] == 0 and n2[i] == 1:
        result.append(1)

    elif n1[i] == 0 and n2[i] == 0:
        result.append(0)

    elif n1[i] == 1 and n2[i] == 1:
        result.append(0)

    elif n1[i] == 1 and n2[i] == 0:
        result.append(1)

for i in result:
    print(i, end="")


"""
THIS IS MORE SIMPLE THAT I DID : (ANSWER #2)

x = input()
y = input()

result = ""
for i in range(len(x)):
    if x[i] == y[i]: result = result + "0"
    else: result = result + "1"

print(result)

"""
